packages/speech/src/index.js

```diff
@@ -0,0 +1,919 @@
+ * @param {object} streamingRecognizeRequest.streamingConfig.config - A
+ *     `RecognitionConfig` object. See [`RecognitionConfig`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig).
+ * @param {string} streamingRecognizeRequest.streamingConfig.config.encoding -
+ *     Encoding of audio data sent in all `RecognitionAudio` messages. See
+ *     [`RecognitionConfig.encoding`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ * @param {number} streamingRecognizeRequest.streamingConfig.config.sampleRate
+ *     - Sample rate in Hertz of the audio data sent in all RecognitionAudio
+ *     messages. See
+ *     [`RecognitionConfig.sampleRate`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ * @param {string=}
+ *     streamingRecognizeRequest.streamingConfig.config.languageCode
+ *     - See [`RecognitionConfig.languageCode`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ *     Default: `"en-US"`.
+ * @param {number=}
+ *     streamingRecognizeRequest.streamingConfig.config.maxAlternatives
+ *     - See [`RecognitionConfig.maxAlternatives`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ *     Default: `1`.
+ * @param {boolean=}
+ *     streamingRecognizeRequest.streamingConfig.config.profanityFilter
+ *     - See [`RecognitionConfig.profanityFilter`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ *     Default: `false`.
+ * @param {string[]=}
+ *     streamingRecognizeRequest.streamingConfig.config.speechContext
+ *     - See [`RecognitionConfig.speechContext`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.RecognitionConfig.FIELDS).
+ *     Default: `[]`.
+ * @param {boolean=} streamingRecognizeRequest.streamingConfig.singleUtterance
+ *     - See [`StreamingRecognitionConfig.singleUtterance`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognitionConfig.FIELDS).
+ *     Default: `false`.
+ * @param {boolean=} streamingRecognizeRequest.streamingConfig.interimResults
+ *     - See [`StreamingRecognitionConfig.interimResults`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognitionConfig.FIELDS).
+ *     Default: `false`.
+ * @returns {object} Readable and writable stream.
+ *
+ * @example
+ * var fs = require('fs');
+ * var through = require('through2');
+ * var toRecognizeRequest = through.obj(function(chunk, encoding, done) {
```

Comment:
What are the use cases for when the user would desire being able to customize the chunk?